internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -1918,4 +1926,20 @@ private ConfigKeys() {
    */
   public static final String CLIENT_PRODUCER_SCHEMA_REFRESH_INTERVAL_SECONDS =
       "client.producer.schema.refresh.interval.seconds";
+
+  /*
+   * The memory up-limit for the ingestion path while using RocksDB Plaintable format.
+   * Currently, this option is only meaningful for DaVinci use cases.
+   */
+  public static final String INGESTION_MEMORY_LIMIT = "ingestion.memory.limit";
+
+  /**
+   * Whether the ingestion is using mlock or not.
```

Comment:
I am confused by the name. Is the config indicating whether the customer's host is using `mlock` or whether Venice ingestion process would be using `mlock`?

Reply:
To customer, they are essentially same thing.

Reply:
Let me clarify this. Our intention is to ask customers to enable it if mlockall is being used so Venice can take this into memory limiter consideration. 
One misleading interpretation of this config is regardless customers has this `mlockall` or not, Venice ingestion process will enable it for them. I could be missing something but I don't think we are doing this in this PR

Reply:
That being said, I've been thinking if there's better naming and feel that `ingestion.mlock.enabled` isn't bad, maybe `application.mlock.enabled`?  Please feel free to drop this comment. Not a big deal anyway.